py/dict.go

```diff
@@ -36,6 +36,29 @@ func init() {
 		}
 		return NewIterator(o), nil
 	}, 0, "items() -> list of D's (key, value) pairs, as 2-tuples")
+
+	StringDictType.Dict["get"] = MustNewMethod("get", func(self Object, args Tuple) (Object, error) {
+		var length = len(args)
+		if length == 0 {
```

Comment:
I believe "Effective Go" recommands to use `switch` for these kind of `if-else-if` code paths:
```go
switch {
case length == 0:
   // ...
case length > 2:
   // ...
}
...
```